ansible-collection-redhatci-ocp.spec

```diff
@@ -51,6 +51,10 @@ find -type f ! -executable -name '*.py' -print -exec sed -i -e '1{\@^#!.*@d}' '{
 
 
 %changelog
+* Fri Jul 19 2024 Ramon Perez <raperez@redhat.com> - 0.13.EPOCH-VERS
+- Version bump due to deprecation of cnf_cert role, moving in favour of
+  k8s_best_practices_certsuite role
+
```

Comment:
Add deprecation of cnf_cert role here

Reply:
It is already in the message: _Version bump **due to deprecation of cnf_cert role**, moving in favour of k8s_best_practices_certsuite role_